overlay/usr/local/src/SHA256SUM

```diff
@@ -4,7 +4,12 @@ f4945bf52ae49da0728fe730a33c18744803752fc948f154f29dc0c4f9f2f9cc  colorama-0.3.7
 67dae9e1af2b0fd71bc9a378654f7dc89211c1c5aee71e160f8cfce1fa6d6980  ecdsa-0.10.tar.gz
 2ab53b434206ed8ae72e9cadb22d44ef9ba720a7d052abe102f5d55cafbef866  Electrum-2.6.4.tar.gz
 093158dbb299b56e98fb760d3721adb43ef7f43b08a792268e1aef454cb0471e  electrum.patch
+876d04757751158df41895dc56dd4823084b41fc2e3d964371432c023aceb515  etherwallet-v3.10.0.2.zip
+845f4dd09f2bbd7131d33ef7b80ec813b75c74040565d33097b5927c67d70815  generateSecretOffline-1.zip
 7f50239d53b5e95b94455d5e1adae70592b5b71f0e960d3bbbfbb125788e6f0b  jsonrpclib-0.1.7.tar.gz
+463ed5ffa13c22f60cc24c4e04a33915720b5f378f4e591c675a1992ca5826b0  liteaddress.org-f31e0d78144d13e2f67142b8b3a67c69c0b2150b.zip
+650ba10126f066daaa6ad204f950949c247f30bcfefeb045bc5b2ce70ac647e6  monero-wallet-generator-c04a4e872bd586ad0d6c1b3d1aa2d9822c1c29f9.zip
```

Comment:
I can confirm that this is trusted by the community. Monero-moo (the developer) is part of Monero's core development team.

To verify:

    wget https://codeload.github.com/moneromooo-monero/monero-wallet-generator/zip/c04a4e872bd586ad0d6c1b3d1aa2d9822c1c29f9
    sha256sum monero-wallet-generator-c04a4e872bd586ad0d6c1b3d1aa2d9822c1c29f9.zip